src/plugins/cosmos/components/modals/Staking/views/UnstakeConfirm.tsx

```diff
@@ -45,6 +53,14 @@ type UnstakeProps = {
 }
 
 export const UnstakeConfirm = ({ assetId, validatorAddress, onCancel }: UnstakeProps) => {
+  const unbondingDays = useMemo(() => {
```

Comment:
we do this memo/effect in 4 places - can we replace this with a function in `StakingCommon`?

```
type AssetIdToUnbondingDays = (assetId: AssetId) => number
```